README.md

```diff
@@ -307,9 +307,8 @@ Lightning is also part of the [PyTorch ecosystem](https://pytorch.org/ecosystem/
 If you have any questions please:
 1. [Read the docs](https://pytorch-lightning.rtfd.io/en/latest/).
 2. [Look it up in our forum (or add a new question)](https://forums.pytorchlightning.ai/)
-2. [Search through the issues](https://github.com/PytorchLightning/pytorch-lightning/issues?utf8=%E2%9C%93&q=my++question).
-3. [Join our slack](https://join.slack.com/t/pytorch-lightning/shared_invite/zt-f6bl2l0l-JYMK3tbAgAmGRrlNr00f1A).
-4. [Ask on stackoverflow](https://stackoverflow.com/questions/ask?guided=false) with the tag pytorch-lightning.
```

Comment:
why remove this one?

Reply:
Because we already have forums + slack where we actively answer questions.

Does anybody visit StackOverflow and answer any open questions? If so, we can close this

Reply:
I don't think anyone https://stackoverflow.com/questions/tagged/pytorch-lightning
Even if someone does, no one is there to answer.